### DISCUSSION TITLE
Android/iPhone access

### DISCUSSION THREAD

**Discussion by jh20001** [zwave-js/zwave-js-ui#1982] [Q&A]
Hey!

This has been absolutely wonderful and I have only been playing with it for a day (using HA).

I was wondering if there are any solutions for accessing this server via a mobile app so you can control the inclusion process and device management without having to be in front of a computer. I can do it using remote desktop, but that's a bit clunky. :)

**Comment by robertsLando** [comment]
>  for accessing this server via a mobile app

You could just open it with your phone browser?

**Reply by jh20001** [reply]
This........is true lol. It doesn't give me access outside of the home, but I guess I wouldn't need it since HA is handling anything I'd need to access remotely. I'll just stick with this idea for now, thanks :)
